app/controllers/resources_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+class ResourcesController < ApplicationController
```

Comment:
We usually add an empty line after the `# frozen_string_literal: true` magic comment. (Rubocop style)